feature/qos/otg_tests/qos_basic_test/README.md

```diff
@@ -94,6 +94,17 @@ Verify that DUT supports QoS config and forward QoS traffic correctly.
 
 *   Verify that there is no traffic loss
 
+*   [TODO] Initiate traffic as per test case 2
```

Comment:
Please add issue # inside [] of TODO. So It will be cleare given TODO is covering specific issue#.
It may happen 2 issues will need 2 separate changes to README simultanously.

Reply:
This is a good practice.  Please create a github issue corresponding to this TODO.  Make sure to set the github lablels "enhancement" and appropriate priority. (p0, p1, p2)

Reply:
Done